lib/saluki-components/src/common/datadog/endpoints.rs

```diff
@@ -205,7 +205,7 @@ impl ResolvedEndpoint {
     }
 
     /// Creates a new  `ResolvedEndpoint` instance from an existing `ResolvedEndpoint`, adding an optional `RefreshableConfiguration`.
-    pub fn with_refreshable_configuration(self, config: Option<RefreshableConfiguration>) -> Self {
+    pub fn with_refreshable_configuration(self, config: Option<GenericConfiguration>) -> Self {
```

Comment:
Seems like we'll also want to remove the "refreshable" configuration wording in this area of the code as well.

Reply:
I changed the wording for now, but I think we may just remove this all together once we actually use the new config subscriber model on the components.